docker-hub/publish/insights-analytics.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Insights & analytics
+description: Provides an API analytics tool for your products.
```

Comment:
We should probably reference the UI that can be used as well as the raw API.

Reply:
Agreed!

Reply:
We should also add screenshots of the UI in this documentation

Reply:
OH yeah. I was totally focused on the API, probably because we merged the API doc updates! Let me add some information about the GUI view.

Reply:
Updated with info about the UI -- please take a look!